README.md

```diff
@@ -19,17 +19,16 @@ To run the MAAP API locally using PyCharm, create a Python Configuration with th
 
 ## II. Local development using python virtualenv
 
-**Pre-requisites:**
+**Prerequisites:**
 
 * postgresql
   * Linux: `sudo apt-get install postgresql python-psycopy2 libpq-dev`
   * Mac OSx: `brew install postgresql`
 * pip, python3.7 and virtualenv
 
 ```bash
-python3 -m venv maap-api-nasa # or whatever environment name you choose
+python3 -m venv venv
 source maap-api-nasa/bin/activate
-# Note: For installing on Mac OSx you may need to replace "psycopg2" with "psycopg2-binary"
```

Comment:
This note is not necessary with a properly configured `requirements.txt` file. See my change to that file.